packages/muze-axis/src/cartesian-axis/continous-axis.js

```diff
@@ -91,14 +92,19 @@ export default class ContinousAxis extends SimpleAxis {
      * @memberof SimpleAxis
      */
     domain (domain) {
-        if (domain && domain.length) {
-            setContinousAxisDomain(this, domain);
-            this.setAxisComponentDimensions();
-            this.logicalSpace(null);
+        if (domain) {
+            if (Array.isArray(domain)) {
```

Comment:
if domain = [], then Array.isArray(domain) will return true. 
Is that an intended result? Previously the check was made for length of the domain.

Reply:
Added check for array length